docs/Working Agreement.md

```diff
@@ -0,0 +1,166 @@
+- Focuses on specific technical challenges needed in project, including deeper technical explorations.
+- Responsible for architecture and leading technical discussions.
+- Works with TPM on feedback, technical gaps, and sharing.
+- Works with TPM to bring in Subject Matter Experts (SMEs) when appropriate.
+
+## Product Owner / Technical Program Manager
+
+- Responsible for turning customer and partner requirements into detailed stories with definitions of done.
+- Responsible for documenting risks appropriately, surfacing risks to the right stakeholders at the right time, and
+  managing mitigations.
+- Responsible for scoping and defining project as mapped to customer/partner business goals.
+- Actively manages the customer Product Owner and executive/business sponsor, and technical stakeholders.
+- Jointly responsible with Tech Lead for Game Plan Documentation and Review.
+- Responsible for backlog organization, grooming, and prioritization with customer/partner.
+- Facilitates customer/partner workshops, leads discussions about the problem and business goals.
+
+## Backlog management
+
+- We communicate what we are working on through the board
+- We assign ourselves a task when we are ready to work on it (not before) and move it to active
+- We capture any work we do related to the project in a user story/task
+- We close our tasks/user stories only when the deliverable has been merged to the code repository
+- We work with the TPM if we want to add a new user story to the sprint
+- If we add a new task to the board, we make sure it matches the acceptance criteria of the user story (to avoid scope
+  creep). If it doesn't match the acceptance criteria we should discuss with the TPM to see if we need a new user story.
+  for the task or if we should adjust the acceptance criteria.
+
+## Estimation
+
+- We will estimate Stories with [Fibonacci
+  Sequence](https://medium.com/ducalis-blog/story-points-how-fibonacci-sequence-works-for-agile-estimation-4c7e0081f4d8)
+  story points to estimate our work, and will make our estimates as a team during Sprint Planning. This will help us
+  gauge how much work we commit to within a sprint.
+
+| Story Points | Estimated Effort                                       |
+|--------------|--------------------------------------------------------|
+| 1pts         | half a day of effort or less                           |
+| 2pts         | full day of effort                                     |
+| 3pts         | day and a half of effort                               |
+| 5pts         | 2 - 3 Days of effort                                   |
+| 8pts         | 4 Days of Effort (~1 week of a sprint)                 |
+| 13pts        | 6 Days of effort (~1.5 weeks of a sprint)              |
+| 21pts        | 10+ Days of effort (a Full end to end Sprint, or more) |
+
+- Tasks and Bugs require hour estimates, as this will track our velocity. Task and bug owners are responsible for
```

Comment:
I prefer to allow more autonomy around this and not do hourly estimates. Especially with a customer this is usually difficult to maintain

Reply:
This was changed in the doc located in the branch issues/37 - Working-Agreement. All updates / changes will be made in that branch